src/utils/setup/fetch-build-time-media.js

```diff
@@ -3,10 +3,11 @@ import { STITCH_AUTH_APP_ID } from '../../constants';
 import { parsePodcasts } from '../parse-podcasts';
 import { simplifyTwitchResponse } from '../simplify-twitch-response';
 import { simplifyYoutubeResponse } from '../simplify-youtube-response';
+import memoizerific from 'memoizerific';
 
 const MAX_RESULTS = 5;
 
-export const fetchBuildTimeMedia = async () => {
+const fetch = async () => {
```

Comment:
```suggestion
const fetchContent = async () => {
```

Just want to be sure we don't overwrite the built-in `fetch` lib.